### PR TITLE
Create Models & Enumerations

### PR DESCRIPTION
**Pull request by SazdovaEkaterina**
- Created the models based on the tables in the database.
- Created enumerations needed.

### CODE REVIEW

File: Prototype Application/Paw5/src/main/java/finki/paw5/model/Food.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "food")
+public class Food {
+
+    @Id
+    @Column(name = "id_food")
```
Comment (FilipChorbeski) [inline Prototype Application/Paw5/src/main/java/finki/paw5/model/Food.java:16]: Needs to be nullable=false

File: Prototype Application/Paw5/src/main/java/finki/paw5/model/PersonalProfile.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "personal_profile")
+public class PersonalProfile {
+
+    @Id
+    @Column(name = "id_pet")
```
Comment (FilipChorbeski) [inline Prototype Application/Paw5/src/main/java/finki/paw5/model/PersonalProfile.java:16]: Needs to be nullable = false

Comment (SazdovaEkaterina) [reply Prototype Application/Paw5/src/main/java/finki/paw5/model/PersonalProfile.java]: [Done.](https://github.com/SazdovaEkaterina/Paw5/pull/3/commits/94308eac1c414decb2696a810dd608e7d83c61c0)

File: Prototype Application/Paw5/src/main/java/finki/paw5/model/Employee.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    }
+
+    public Employee(int id, String position, int shelterId, boolean verified, int verifiedByAdminId) {
+        this.id = id;
+        this.position = position;
+        this.shelterId = shelterId;
+        this.verified = verified;
+        this.verifiedByAdminId = verifiedByAdminId;
+    }
```
Comment (trajchevaM) [inline Prototype Application/Paw5/src/main/java/finki/paw5/model/Employee.java:40]: Дали посвојувачи, донори... да наследуваат од user?

File: Prototype Application/Paw5/src/main/java/finki/paw5/model/Pet.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "pet")
+public class Pet {
+
+    @Id
```
Comment (trajchevaM) [inline Prototype Application/Paw5/src/main/java/finki/paw5/model/Pet.java:18]: nullable = true

**Review by FilipChorbeski — DISMISSED** [review]

File: Prototype Application/Paw5/src/main/java/finki/paw5/model/Pet.java (hunk 4 above)
Comment (SazdovaEkaterina) [reply Prototype Application/Paw5/src/main/java/finki/paw5/model/Pet.java:18]: [Done.](https://github.com/SazdovaEkaterina/Paw5/pull/3/commits/2fdea25b8110aa31f0d6fe005d9488c25f8799ad)

File: Prototype Application/Paw5/src/main/java/finki/paw5/model/User.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    @Column(name = "name_user", nullable = false, length = 100)
+    private String name;
+
+    @Column(name = "email_user", nullable = false, length = 100, unique=true)
+    private String email;
+
+    @Column(name = "password_user", nullable = false, length = 20)
+    private String password;
+
+    @Column(name = "telephone", nullable = false, length = 20)
```
Comment (trajchevaM) [inline Prototype Application/Paw5/src/main/java/finki/paw5/model/User.java:32]: telephone mozhe da e nullable

Comment (SazdovaEkaterina) [reply Prototype Application/Paw5/src/main/java/finki/paw5/model/User.java]: [Done.](https://github.com/SazdovaEkaterina/Paw5/pull/3/commits/e1642f4a8de4fb873993f0e11f437c01e0d4a143)

**Review by FilipChorbeski — DISMISSED** [review]

**Comment by SazdovaEkaterina** [comment]
TODO: Resolve data source issue.

**Review by SazdovaEkaterina — COMMENTED** [review]
TODO: Resolve data source issue.

File: Prototype Application/Paw5/src/main/java/finki/paw5/model/Employee.java (hunk 3 above)
Comment (SazdovaEkaterina) [reply Prototype Application/Paw5/src/main/java/finki/paw5/model/Employee.java:40]: Will be fixed with[ inheritance annotation](https://www.baeldung.com/hibernate-inheritance).

Comment (SazdovaEkaterina) [reply Prototype Application/Paw5/src/main/java/finki/paw5/model/Employee.java]: [Done.](https://github.com/SazdovaEkaterina/Paw5/pull/3/commits/579bf6dee959bcd85e613fae39f3e9dd5498d4c0)

**Review by FilipChorbeski — APPROVED** [review]

**Pull request merged**